botwo/botwo.py

```diff
@@ -1,131 +1,144 @@
 import fnmatch
+import botocore
 
-SPECIAL_METHODS = {"get_paginator", "delete_objects"}
 COPY_METHODS = {"copy", "copy_object", "copy_upload_part"}
+LIST_METHODS = {"list_objects", "list_objects_v2", "list_object_version"}
 
 
-def client(client_mapping, config):
-    botwo = Botwo()
-    return botwo.create_client_class(client_mapping, config)
+class PaginatorWrapper(object):
+    """ This class wraps boto paginator """
 
-
-class Paginatwo(object):
-
-    def __init__(self, mapping, kwargs, client1, client2, client1_name, client2_name):
+    def __init__(self, mapping, config, kwargs):
         self.mapping = mapping
-        self.client1_name = client1_name
-        self.client2_name = client2_name
-        self.paginator1 = client1.get_paginator(kwargs["operation_name"])
-        self.paginator2 = client2.get_paginator(kwargs["operation_name"])
+        self.config = config
+        self.paginators = dict()
+        for client in self.mapping:
+            self.paginators[client] = self.mapping[client].get_paginator(kwargs["operation_name"])
```

Comment:
If this function took *two* args `operation_name` and `**kwargs`, then it could simply _ignore_ `kwargs` and access `operation_name` directly.

Reply:
Done